datura/protocol.py

```diff
@@ -459,6 +459,29 @@ def extract_json_chunk(chunk, response, buffer=""):
     return json_objects, buffer
 
 
+class SearchSynapse(bt.Synapse):
```

Comment:
make sure you add comments in this class that shows how to use it as an API, such as what are the available tools, and explaining what each tool will do. This is a common difficulty I have with other subnets, it's unclear how to use them. So please add as much comments as possible to the full protocol, not just this class.

Reply:
Okay.
We also have Swagger docs for search. Docs can be accessed by `/docs` endpoint


<img width="1280" alt="image" src="https://github.com/Datura-ai/smart-scrape/assets/22216909/4ba75056-4134-4bd9-a090-b68fc106df95">